src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <section class="calendar calendar--start-day-mon calendar--month-length-31">
```

Comment:
start-day and month length modificators should look like this `calendar--start-day--mon` according to BEM